src/components/SideMenu/index.jsx

```diff
@@ -0,0 +1,46 @@
+import React, { useContext } from 'react';
+import { Link } from 'react-router-dom';
+import { AUTH_STORAGE_KEY } from '../../utils/constants';
+import { storage } from '../../utils/storage';
+import { LinkText, ContainerMenu, SideMen, CloseBtn, Links, CloseIcon } from './styled';
+import ThemeContext from '../../state/ThemeContext';
+
+const SideMenu = ({ open, setOpen }) => {
+  const isLogged = storage.get(AUTH_STORAGE_KEY);
+  const { stateTheme } = useContext(ThemeContext);
+  const { theme } = stateTheme;
+
+  const favorites = (
+    <Link to={{ pathname: `/` }} onClick={() => setOpen(!open)}>
+      <LinkText theme={theme}>
+        <span role="img" aria-label="favorites">
+          ⭐️
+        </span>
+        Favorites
+      </LinkText>
+    </Link>
+  );
```

Comment:
Quick suggestion for readability:

```suggestion
  const Favorites = ({ isLogged } ) => isLogged 
  ? (
    <Link to={{ pathname: `/` }} onClick={() => setOpen(!open)}>
      <LinkText theme={theme}>
        <span role="img" aria-label="favorites">
          ⭐️
        </span>
        Favorites
      </LinkText>
    </Link>
  )
  : null ;
```

Then, down below where you're using it you just provide more JSX as in:
```js
<Favorites isLogged={isLogged} />
```

In that way Favorites could be extracted into a separate file.